roles/frontend/handlers/main.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- name: start frontend.service
+  service:
+    name: frontend.service
+    state: started
```

Comment:
I believe you mean `restarted` here, don't you?

Reply:
yes